sdk/confidentialledger/azure-security-confidentialledger/pom.xml

```diff
@@ -34,6 +34,7 @@
     <jacoco.skip>true</jacoco.skip>
     <javaModulesSurefireArgLine>
       --add-exports com.azure.core/com.azure.core.implementation.jackson=ALL-UNNAMED
+      --add-exports com.azure.http.netty/com.azure.core.http.netty=com.azure.security.confidentialledger
     </javaModulesSurefireArgLine>
   </properties>
 
```

Comment:
Do the pom, version_client, and changelog files need to be updated for the GA version or would be done in a follwo up PR?

Reply:
I asked Weidong about a pom.xml change and he said the following:
"On release, it should be changed to 1.0.0
But that would be a separate release PR for it. So don't change it now.
https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/8/Release-Checklist at "Prepare Release Script"
that is after this PR."

For additional context, my question was:
"Should I change this?
<groupId>com.azure</groupId>
  <artifactId>azure-security-confidentialledger</artifactId>
  <version>1.0.0-beta.3</version>
I was going to set the version to 1.0.1"

Reply:
It should be fine to change it in a follow-up PR. And for GA release, it should be changed to 1.0.0.